docs/fsharp/language-reference/literals.md

```diff
@@ -79,11 +76,23 @@ In pattern matching expressions, identifiers that begin with lowercase character
 
 Signed 32-bit integers can also be specified in hexadecimal, octal, or binary using a `0x`, `0o` or `0b` prefix respectively.
 
-```
+```fsharp
 let numbers = (0x9F, 0o77, 0b1010)
 // Result: numbers : int * int * int = (159, 63, 10)
 ```
 
+## Underscores in Numeric Literals
+
+Starting with F# 4.1, you can separate digits with the underscore character (`_`).
+
+```fsharp
+let deadBeef = 0xDEAD_BEEF
+
+let deadBeefAsBits = 0b1101_1110_1010_1101_1011_1110_1010_1111
+
+let ssn = 012_345_6789
```

Comment:
Does F# assign a special meaning when the first digit of a numeric literal is 0? If not, you may want to change the value used in this example.